### DISCUSSION TITLE
How do you get the grid to stay on the map?

### DISCUSSION THREAD

**Discussion by Daza99** [luiscastilho/dungeoneering#11] [Q&A] [answered]
I worked out how to do the grid, but as soon as you click on something else it disappears off the map. I seemed to have worked it out. I had to click on the Grid Setup Icon again once I have it set to my liking. This bit of info could be added to the text instructions when you first click on it. 

Also only when is the grid is done will the Add Player and DM token buttons become active once the grid is set, which makes sense since they will be scaled according to grid size you set.

**Comment by luiscastilho** [comment, marked as answer]
>  I had to click on the Grid Setup Icon again once I have it set to my liking.

Yes, you have to click on the Grid Setup icon again once you are done.

> This bit of info could be added to the text instructions when you first click on it.

Great suggestion! I will add this to the instructions.

> Also only when is the grid is done will the Add Player and DM token buttons become active once the grid is set, which makes sense since they will be scaled according to grid size you set.

Exactly. Grid setup is enabled after you have selected a map. Walls, doors and tokens are enabled once you have set up a grid.

I did a [video showing how to use dungeoneering](https://www.youtube.com/watch?v=mLLleHoVkdk). It's linked in the main page, under How to Use It. I imagine it's slightly outdated by now, but should still be useful.
